frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -414,14 +414,11 @@ class Transaction extends Component<Props, State> {
               </div>
             </div>
             <div className={[style.detail, 'flex-center'].join(' ')}>
-              <p>
-                <A
-                  className={style.externalLink}
-                  href={explorerURL + transactionInfo.txID}
-                  title={t('transaction.explorerTitle') + '\n' + explorerURL + transactionInfo.txID}>
-                  {t('transaction.explorerTitle')}
-                </A>
-              </p>
+              <A
+                href={explorerURL + transactionInfo.txID}
+                title={t('transaction.explorerTitle') + '\n' + explorerURL + transactionInfo.txID}>
```

Comment:
nit: Following our modern-js pattern we can concatenate this using string templating, smt like this:

```js
 `${t(something.title)} \n ${explorerURL}${transactionInfo.txID}`
```

Reply:
done